src/main/java/StringLinks.java

```diff
@@ -0,0 +1,44 @@
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class StringLinks implements Links<String>{
+
+    protected String url;
+
+    public StringLinks(String url) {
+        this.url = url;
+    }
+
+    private List<String> prepareLinks() {
```

Comment:
@PavelNejinskiy please use this convention while naming methods/classes
http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html